CreateInstaller.ps1

```diff
@@ -0,0 +1,12 @@
+$tempDirectoryName = '.\__squirrel_temp__'
+
+if ($DTE -eq $null) {
+  echo "Run this from the NuGet Package Console inside VS"
+  exit 1
+}
+
+mkdir -Path $tempDirectoryName
+rm -r -fo "$tempDirectoryName\*.nupkg"
+NuGet pack .\src\AudioSwitcher\AudioSwitcher.csproj -OutputDirectory "$tempDirectoryName" -Prop Configuration=Release
```

Comment:
Ran into the same problem with Squirrel, but look like this is because the package was _just_ restored. If I restarted VS, then the script worked. Looks like a NuGet bug, filed: https://nuget.codeplex.com/workitem/4439#